starky/src/f3g.rs

```diff
@@ -47,7 +91,7 @@ impl F3G {
     }
 
     #[inline]
-    pub fn mul_scalar(self, b: usize) -> Self {
+    fn mul_scalar(self, b: usize) -> Self {
```

Comment:
Here should pass the reference instead of the ownship of it.

Reply:
Hi, here still passing the ownership `self`. we'd better pass its reference `&self`